sdk/communication/arm-communication/README.md

```diff
@@ -63,7 +63,17 @@ const client = new CommunicationServiceManagementClient(new DefaultAzureCredenti
 // });
 // const client = new CommunicationServiceManagementClient(credential, subscriptionId);
 ```
-
+```javascript
+const resource = {
+    location: "Global",
+    dataLocation: "United States",
+    identity: {
+        type: "SystemAssigned"
```

Comment:
Also, what about user assigned?

Reply:
added a explanation for the user assigned